timur/lib/client/jsx/components/browser/browser.jsx

```diff
@@ -102,7 +101,7 @@ export default function Browser({ model_name, record_name, tab_name }) {
     } else if (!tab_name) {
       selectDefaultTab(view);
     } else {
-      showTab();
+      showTab(view);
```

Comment:
You were correct -- the view needs to be passed it in.  It worked before by using the view closure from the state, which is out of date when this function is invoked as part of some other callback that has already updated the state.